python/tests/res/enkf/test_model_config.py

```diff
@@ -163,3 +163,15 @@ def test_model_config_dict_constructor(self):
                                        refcase=res_config.ecl_config.getRefcase(),
                                        config_dict=config_dict)
             self.assertEqual(model_config, res_config.model_config)
+
+    def test_schedule_file_as_history_is_disallowed(self):
+        case_directory = self.createTestPath("local/configuration_tests")
+        with TestAreaContext("test_constructor") as work_area:
+            work_area.copy_directory(case_directory)
+            with self.assertRaises(ValueError) as cm:
+                ResConfig(
+                    user_config_file="configuration_tests/sched_file_as_history_source.ert"
+                )
+            self.assertIn(
```

Comment:
shouldnt this be on the same indentation level?

Reply:
No, then any exception raised by `self.assertIn` is eaten by the `with-block`.